shotover-proxy/tests/test-configs/cassandra/cluster-multi-rack-2-per-rack/docker-compose.yaml

```diff
@@ -11,7 +11,7 @@ networks:
 
 services:
   cassandra1_1:
-    image: &image library/cassandra:4.0.6
+    image: &image shotover/cassandra-test:4.0.6-r1
```

Comment:
Converting this to use shotover/cassandra-test is probably worth doing but it will copying the tokens from a `SELECT * FROM system.local; SELECT * FROM system.peers;` of a running `library/cassandra:4.0.6` cluster and then adding them to the CASSANDRA_INITIAL_TOKENS env var.